[2023-12-03 18:24:04,528] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_automation_daily.testing manual__2023-12-03T18:24:00.682672+00:00 [queued]>
[2023-12-03 18:24:04,583] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_automation_daily.testing manual__2023-12-03T18:24:00.682672+00:00 [queued]>
[2023-12-03 18:24:04,589] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-12-03 18:24:04,589] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-12-03 18:24:04,590] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-12-03 18:24:04,670] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): testing> on 2023-12-03 18:24:00.682672+00:00
[2023-12-03 18:24:05,019] {standard_task_runner.py:52} INFO - Started process 651 to run task
[2023-12-03 18:24:05,039] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_automation_daily', 'testing', 'manual__2023-12-03T18:24:00.682672+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsir4uef7', '--error-file', '/tmp/tmp_rk2mxx6']
[2023-12-03 18:24:05,041] {standard_task_runner.py:80} INFO - Job 46: Subtask testing
[2023-12-03 18:24:05,555] {task_command.py:369} INFO - Running <TaskInstance: dag_automation_daily.testing manual__2023-12-03T18:24:00.682672+00:00 [running]> on host c9140d11a0a8
[2023-12-03 18:24:05,887] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sebastiancahyoardhiiswara@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_automation_daily
AIRFLOW_CTX_TASK_ID=testing
AIRFLOW_CTX_EXECUTION_DATE=2023-12-03T18:24:00.682672+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-03T18:24:00.682672+00:00
[2023-12-03 18:24:05,891] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-03 18:24:05,898] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python codes/test.py']
[2023-12-03 18:24:05,929] {subprocess.py:85} INFO - Output:
[2023-12-03 18:24:08,618] {subprocess.py:92} INFO - mysql+mysqlconnector://root:mysql@db_mysql_staging:3306/mysql
[2023-12-03 18:24:08,618] {subprocess.py:92} INFO - postgresql://***:***@db_postgres_dwh:5432/***
[2023-12-03 18:24:08,619] {subprocess.py:92} INFO - None
[2023-12-03 18:24:08,619] {subprocess.py:92} INFO - user: postgres
[2023-12-03 18:24:08,620] {subprocess.py:92} INFO - password: postgres
[2023-12-03 18:24:08,620] {subprocess.py:92} INFO - db: dwh
[2023-12-03 18:24:08,745] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-03 18:24:08,794] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_automation_daily, task_id=testing, execution_date=20231203T182400, start_date=20231203T182404, end_date=20231203T182408
[2023-12-03 18:24:08,849] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-03 18:24:08,968] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
